app/assets/javascripts/datarepo/funder.js

```diff
@@ -0,0 +1,34 @@
+$(document).ready(function(){
+
+  $('.collection_funder').find('.collection_funder').each( function(index, value) {
+    xmlDoc = $.parseXML( value.value ),
+    $xml = $( xmlDoc ),
+      $fundername = $xml.find( "fundername" );
+      $awardnumber = $xml.find( "awardnumber" );
+      if ($fundername.length > 0 ) {
+        newstring = $fundername.text() + ":" + $awardnumber.text();
+        $('.collection_funder').find('.collection_funder')[index].value = newstring;
+      }
+  });
+
+  $('#create_submit, #update_submit').click(function( event ) {
```

Comment:
Adding a class helps here too, as in https://github.com/VTUL/data-repo/blob/master/app/assets/javascripts/datarepo/autocomplete.js#L49